tests/wallet/cat_wallet/test_cat_wallet.py

```diff
@@ -23,12 +23,6 @@
 from tests.time_out_assert import time_out_assert
 
 
-@pytest.fixture(scope="module")
-def event_loop():
```

Comment:
some of these are necessary, for now (see comment in `test_mempool.py`). But I think a fair number of them have just been copy-pasted into new tests for no reason